egs/wsj/s5/steps/nnet3/lstm/make_configs.py

```diff
@@ -104,6 +106,8 @@ def ParseLstmDelayString(lstm_delay):
                         help="dimension of non-recurrent projection")
     parser.add_argument("--hidden-dim", type=int,
                         help="dimension of fully-connected layers")
+    parser.add_argument("--preserve-state", type=str,
```

Comment:
<code>if true, additional nodes necessary for state preserving RNN training are created. The states of the recurrent nodes in the previous minibatch are preserved for the next minibatch.</code>